test/ui/ui_unit.js

```diff
@@ -409,6 +409,89 @@ describe('UI', () => {
       });
     });
 
+    describe('control panel buttons with submenus', () => {
+      /** @type {!HTMLElement} */
+      let resolutionMenu;
+      /** @type {!string} */
+      let resolutionDisplay;
+      /** @type {!Element} */
+      let resolutionMenuButton;
+      /** @type {!HTMLElement} */
+      let languageMenu;
+      /** @type {!string} */
+      let languageDisplay;
+      /** @type {!Element} */
+      let languageMenuButton;
+
+      beforeEach(() => {
+        const config = {
+          controlPanelElements: [
+            'quality',
+            'language',
+          ],
+        };
+        const ui = UiUtils.createUIThroughAPI(videoContainer, video, config);
+        player = ui.getControls().getLocalPlayer();
+
+        const resolutionsMenus =
+        videoContainer.getElementsByClassName('shaka-resolutions');
+        expect(resolutionsMenus.length).toBe(1);
+        resolutionMenu = /** @type {!HTMLElement} */ (resolutionsMenus[0]);
+        resolutionDisplay =
+        window.getComputedStyle(resolutionMenu, null).display;
+
+        const resolutionMenuButtons =
+        videoContainer.getElementsByClassName('shaka-resolution-button');
+        expect(resolutionMenuButtons.length).toBe(1);
+        resolutionMenuButton = resolutionMenuButtons[0];
+
+        const languageMenus =
+        videoContainer.getElementsByClassName('shaka-resolutions');
```

Comment:
Is it still 'shaka-resolutions' for the language menus?

Reply:
Thank you Michelle!

I realized that having this typo meant there was an underlying issue, which lead me to find out that the `event` on the listener that hides menus on `ui/controls.js` could have different `targets` that should display the same behavior.

Fixed the implementation on my last commit!